### PR TITLE
Fix/build

### PR DESCRIPTION
**Pull request by makosblade**
This PR:
- fixes build.gradle code
- updates versions 
- update kotlin, spotbugs configuration

### CODE REVIEW

**Comment by katherinebernstein** [comment]
(side note that I don't know how copyrights work but these added dates are 2018/2015 -- just wanted to sanity check that)

File: build.gradle [hunk 1]
```diff
@@ -155,16 +188,6 @@ install {
...
     ext.downloadSources = true
     ext.downloadJavadoc = true
-    ext.sourceCompatibility = JavaVersion.VERSION_1_8
-}
 
-compileKotlin {
-    kotlinOptions {
-        jvmTarget = "1.7"
-    }
+    ext.sourceCompatibility = JavaVersion.VERSION_11
```
Comment (geekbeast) [inline build.gradle:192]: Not sure this is safe.

Comment (UnsungHero97) [reply build.gradle:192]: It'll very likely break things for older devices. Let's revert.

File: build.gradle [hunk 2]
```diff
@@ -27,31 +48,40 @@ ext.connectionUrl = 'scm:git:https://github.com/openlattice/chronicle-api.git'
 ext.developerConnectionUrl = 'scm:git:https://github.com/openlattice/chronicle-api.git'
 
 apply from: "https://raw.githubusercontent.com/openlattice/gradles/master/openlattice.gradle"
-apply plugin: 'kotlin'
-apply plugin: 'jacoco'
+
+sourceCompatibility = JavaVersion.VERSION_11
+targetCompatibility = JavaVersion.VERSION_11
```
Comment (UnsungHero97) [inline build.gradle:53]: Please revert to `JavaVersion.VERSION_1_8`.

**Comment by makosblade** [comment]
Switched back to Java 8, added a comment

**Pull request merged**